src/lib/widgets/now-playing/ui/now-playing.svelte

```diff
@@ -30,7 +46,27 @@
         <EpisodeDisplay {episode} {podcast} />
       </div>
     </div>
-    <FullControls class="w-full mt-5" />
+    <ScrubbingBar
+      duration={$duration}
+      position={$currentTime}
+      buffered={$buffered}
+      on:scrub={(e) => {
+        if (e.detail.dragging) pause();
+        else play();
+        seek(e.detail.position);
+      }}
+    />
+    <div class="w-full mt-5 flex justify-around">
+      <button
+        name="Toggle playback speed"
```

Comment:
issue (non-blocking): the action is not really "toggle", more like "cycle", as in the function name. I'm thinking, maybe we could even write the next speed in the button name? Only do it if you feel like doing it, it's a lot of code for little benefit